acts

```diff
@@ -149,7 +149,7 @@ archives=$($tarsnap --list-archives | sort -n)
 # PART 4: Trim old backups.
 
 if [ "$backuprc" != "0" ] ; then
-    log_message "One of the backups failed -- not deleting old backups"
+    log_verbose "at=error type=$archivetype"
```

Comment:
Yeah agreed. Just going to revert this change. The same information is already logged with more context above:

https://github.com/alexjurkiewicz/acts/pull/8/files#diff-fa1679bbd1d49c770b4ff7d0074bc24eR139